x-pack/solutions/search/test/functional_search/config.ts

```diff
@@ -41,6 +41,15 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       searchInferenceEndpoints: {
         pathname: '/app/elasticsearch/relevance/inference_endpoints',
       },
+      searchOverview: {
+        pathname: '/app/elasticsearch/overview',
+      },
+      searchHomepage: {
+        pathname: '/app/elasticsearch/home',
+      },
+      searchGettingStarted: {
```

Comment:
I defined this in the base config instead of the feature flag config because it will need to be added here anyway once the FF is removed